TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Actively_Active_Intake.java

```diff
@@ -8,44 +8,46 @@
 import com.qualcomm.robotcore.hardware.HardwareMap;
 import com.qualcomm.robotcore.hardware.Servo;
 
+import org.firstinspires.ftc.robotcore.external.Telemetry;
+
 @TeleOp(name="BensonIntake", group="Linear OpMode")
 public class Actively_Active_Intake extends LinearOpMode {
 
     Servo rotate;
     CRServo spin;
 
     void initialize(HardwareMap hwm){
-        Servo rotate = hwm.get(Servo.class, Constants.BENSON_INTAKE_ROTATE);
+        rotate = hwm.get(Servo.class, Constants.BENSON_INTAKE_ROTATE);
         rotate.setDirection(Servo.Direction.FORWARD);
-        CRServo spin = hwm.get(CRServo.class, Constants.BENSON_INTAKE_SPIN);
+        spin = hwm.get(CRServo.class, Constants.BENSON_INTAKE_SPIN);
         spin.setDirection(DcMotorSimple.Direction.FORWARD);
     }
-    void rotateOut(){
+    void rotateOut(Telemetry telemetry){
         rotate.setPosition(135);
         telemetry.addData("Intake Position", "%4.2f", rotate.getPosition());
     }
-    void rotateIn(){
+    void rotateIn(Telemetry telemetry){
         rotate.setPosition(90);
         telemetry.addData("Intake Position", "%4.2f", rotate.getPosition());
     }
-    void startSpin(){
+    void startSpin(Telemetry telemetry){
         spin.setPower(1);
         telemetry.addData("Intake Power", "%4.2f", spin.getPower());
     }
-    void stopSpin(){
+    void stopSpin(Telemetry telemetry){
         spin.setPower(0);
         telemetry.addData("Intake Power", "%4.2f", spin.getPower());
     }
```

Comment:
Something to think about - we'll need to be able to spin in the opposite direction as well to 'unload' the sample.